templates/citizen_project/media_generator_image_form.html.twig

```diff
@@ -0,0 +1,132 @@
+{% extends 'citizen_project/media_generator.html.twig' %}
+
+{% block page_title 'Projets citoyens - image generateur' %}
+
+{% block media_generator_form %}
+    <p>Grâce a cette interface, generez les photos du profil et de couverture de votre projet citoyen sur les reseaux sociaux.</p>
+
+    {{ form_start(form) }}
+
+    {{ form_errors(form) }}
+
+    <div class="l__row l__row--center l__row--v-stretch b__nudge--bottom-60">
+
+        <div class="l__col l__col--half l__col--top b__nudge--right-larger">
+
+            <div class="img-gen__fieldset">
+                {{ form_row(form.citizenProjectTitle, {
+                    label: 'Entrez le nom de votre projet citoyen :',
+                    attr: {class: 'img-gen__input img-gen__input--block'},
+                    label_attr: {class: 'img-gen__label'}
```

Comment:
There should be spaces inside brackets: `label_attr: { class: 'img-gen__label' }`

but i notice it's the same everywhere on this PR, not a big deal.